extensions/reqbin-zip/src/constants/url.ts

```diff
@@ -0,0 +1 @@
+export const url = process.env.NODE_ENV === "development" ? "http://localhost:3000" : "https://reqbin.zip";
```

Comment:
Other than you, would any user benefit from having localhost here? It seems a bit odd having it in prod.